plugins/source/gcp/policies_v1/views/buckets_permissions.sql

```diff
@@ -1,15 +1,15 @@
--- CREATE OR REPLACE VIEW gcp_public_buckets_accesses AS
--- WITH project_policy_roles AS (SELECT project_id,
---                                      name,
---                                      self_link,
---                                      jsonb_array_elements(p.policy -> 'bindings') AS binding
---                               FROM gcp_storage_buckets p),
---      role_members AS (SELECT project_id,
---                              name,
---                              self_link,
---                              binding ->> 'role'                              AS "role",
---                              jsonb_array_elements_text(binding -> 'members') AS MEMBER
---                       FROM project_policy_roles)
--- SELECT project_id, name, self_link, "role", MEMBER
--- FROM role_members
--- TODO
\ No newline at end of file
+CREATE OR REPLACE VIEW gcp_public_buckets_accesses AS
+WITH project_policy_roles AS (SELECT p.project_id,
+                                     p.name,
+                                     CASE WHEN p.name like '%.appspot.com' THEN 'https://'||p.name ELSE 'https://' || p.name || '.storage.googleapis.com' END AS self_link,
```

Comment:
This is my attempt at generating the `self_link` since it's not in BucketAttrs any more... or maybe just remove this column